extension/src/popup/components/sendPayment/SendTo/index.tsx

```diff
@@ -54,13 +54,11 @@ export const shouldAccountDoesntExistWarning = (
 
 export const AccountDoesntExistWarning = () => {
   const { t } = useTranslation();
+  const notificationTitle = t("The destination account doesn’t exist");
```

Comment:
ah think you just need to merge master. Think there's a small diff issue here

Reply:
Hmm I did merge to master 🤔 let me see why this diff still exists.